### PR TITLE
Updated what's new for 0.12

### PR DESCRIPTION
**Pull request by pelson**
This What's New isn't quite complete, but I wanted to get a bit of visibility (and somebody to merge it if they are keen - I can always make a smaller PR later which finishes the job).

I think you will agree when you look at the ``what's new``, that v0.12 represents a huge amount of effort from many people, not least @SciTools/cartopy-devs, so a big thank you to everybody involved!

Based on his really great work, I'd like to nominate @QuLogic as our newest core developer. @QuLogic - if your workload means that you can't realistically accept the nomination, we would understand, but if you'd like to accept it would be great to have you on the team! For the benefit of @SciTools/cartopy-devs, I've put together a little note about some of @QuLogic's great work recently.

Finally, I plan to cut a release tomorrow. Because there is so much in here, I intend to cut a release candidate before cutting the actual tagged release around the 3rd of March.

### CODE REVIEW

File: docs/source/whats_new.rst [hunk 1]
```diff
@@ -1,11 +1,114 @@
...
+                                 central_rotated_longitude=180)
+        ax = plt.axes(projection=rpole)
+        ax.set_global()
+        ax.gridlines()
+        ax.stock_img()
+        ax.coastlines()
+        plt.show()
+
+* A new method has been added to the :class:`~cartopy.mpl.geoaxes.GeoAxes` to
+  allow control the neatline of a map drawn with the matplotlib interface.
```
Comment (rhattersley) [inline docs/source/whats_new.rst]: ... control **of** the ... ?

**Comment by rhattersley** [comment]
> I'd like to nominate @QuLogic as our newest core developer

Damn! You beat me to it! That thought has been banging around my head for a couple of weeks now ... probably trying to find a neuron to bump into. :wink: 

I'm :+1: :+1: in favour. Thanks @QuLogic!

**Comment by rhattersley** [comment]
This PR is looking good ... top work @pelson! :+1:

**Comment by ajdawson** [comment]
>I'd like to nominate @QuLogic as our newest core developer

:+1: great idea!

**Comment by QuLogic** [comment]
Whoa! :astonished:
Happy to be here! :bow:

File: docs/source/whats_new.rst [hunk 2]
```diff
@@ -1,11 +1,114 @@
...
+  :class:`~cartopy.crs.AlbersEqualArea` have been added. See the :ref:`cartopy_projections`
+  for the full list of projections now available in cartopy.
+
+* The Web Map Service (WMS) interface has been extended to support on-the-fly reprojection
+  of imagery if the service does not support the projection of the map being drawn.
+
+    .. plot:: examples/wms.py
+       :width: 200pt
+
+* Peter Killick added an interface for accessing MapBox tiles using the MapBox
```
Comment (pelson) [inline docs/source/whats_new.rst]: Ping @dkillick.

File: docs/source/whats_new.rst [hunk 3]
```diff
@@ -1,11 +1,114 @@
...
+  interface for another source of imagery:
+
+    .. plot:: examples/image_tiles.py
+       :width: 200pt
+
+* Some improvements were made to the geometry transformation algorithm to improve
+  the stability of geometry winding. Several cases of geometries being incorrectly
+  inverted when transformed have now been resolved. (:pull:`545`)
+
+* Mark Hedley added the ``central_rotated_longitude`` keyword to
```
Comment (pelson) [inline docs/source/whats_new.rst]: Ping @marqh.

**Comment by pelson** [comment]
> Whoa! :astonished:
> Happy to be here! :bow:

I take that as an acceptance! :+1: 
I've added you on to the cartopy-dev team. Really looking forward to more of your feature contributions, as well as getting involved with reviews and welcoming newcomers to the codebase.

Congratulations! :tada:

**Comment by rhattersley** [comment]
> I've added you on to the cartopy-dev team

@QuLogic - Your membership is currently "private" so doesn't show up at https://github.com/SciTools in the People box, or at https://github.com/SciTools/people. I believe only you can make it public if that's what you'd prefer.

**Comment by QuLogic** [comment]
> I believe only you can make it public if that's what you'd prefer.

Ah yes. Fixed.

**Comment by pelson** [comment]
Thanks @QuLogic - and congrats on your first merge! :smile:

**Pull request merged**